HomePage.html

```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
 						</ul>
 
 					<button class="scroll-button">
-						<img src="Assets/Icons/scroll-button.svg" alt=""/>
+						<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt=""/></a>
 					</button>
 		
 				</section>
 	
+		</div>
+
+		
+		<div class="project-page" id="projects">
+			
+			<section >
```

Comment:
- [ ] [REQUIRED] The section tag should be enough to wrap the about section. If you really want to wrap it with two tags then let the section be the outer tag.

Reply:
@fobadara Ive switched it up to make the parent a section element with two divs inside.

Reply:
I was trying to explain that the section tag is enough as a divider between the different parts of your site so you don't need to wrap the section tags in a div tag